### DISCUSSION TITLE
how to conver char array struct

### DISCUSSION THREAD

**Discussion by wenfengtou** [nlohmann/json#2810] [Q&A] [answered]
```cpp
typedef struct
{
    char userName[10];   
    int age
} ProxyConfUserInfo;
```

hello,how to conver char array struct like this?

**Comment by nlohmann** [comment, marked as answer]
Char arrays can be converted like this:

```cpp
char username[] = "hello";
json j = username;
```

Read anything else on user-defined type conversion here: https://github.com/nlohmann/json#arbitrary-types-conversions

**Reply by nlohmann** [reply]
Do you need further assistance with this issue?
